Python/correlation/test/test_vary_frames.py

```diff
@@ -59,24 +58,19 @@ def test_plot_of_signals_and_correlation_signal():
 
     correlation_matrix, lag_matrix = correl.calculate_correlation_matrix(cut_result)
 
-    #  Not necessary for test, but nice for debugging.
-    tdoa_array = tdoa.calculate_tdoa_array(correlation_matrix, lag_matrix)
-    differance_tdoa_array = tdoa.find_diff_between_tdoa_array(
-        hydro_array=hydro_array,
-        tdoa_array=tdoa_array,
-        pinger_position=pinger.position,
-        sampling_frequency=pinger.sampling_frequency,
-    )
-
```

Comment:
@hannahcl I took the liberty of removing this, as we hopefully don't have to debug it anymore at this point.